model/TrajectoryDataModel.sql

```diff
@@ -20,20 +20,20 @@ $BODY$
 DECLARE
   t trajectory;
 BEGIN
-    t.geom_type = t_type($1);
+    /*t.geom_type = t_type($1);
```

Comment:
@FxKu You just need to rename it to `tg_type` and It should work. Seems like we renamed `t_type` to `tg_type` but never commit the change here.